core/policy_test.go

```diff
@@ -73,7 +75,10 @@ func TestMessageQueuePolicy(t *testing.T) {
 			b.IncHeight(103)
 		})
 		b1 := blocks.BuildOn(root, func(b *chain.BlockBuilder) {
-			b.AddMessage(msgs[0], &types.MessageReceipt{})
+			b.AddMessages(
+				[]*types.SignedMessage{msgs[0]},
+				types.EmptyReceipts(1),
+			)
```

Comment:
Even though it's clumsy code in the builder, it would be better if the old `b.AddMessage` interface were retained for users to avoid the noise of the slice literal construction and the possibility of non-parallel slices.